### DISCUSSION TITLE
Lack of control via keys

### DISCUSSION THREAD

**Discussion by hardglitch** [BLumia/pineapple-pictures#35] [Ideas]
Strongly lack keys to image scrolling (left arrow and right arrow)

**Comment by BLumia** [comment]
Hi, you can use <kbd>PageUp</kbd> and <kbd>PageDown</kbd> to navigate between the previous and the next images. The <kbd>Forward</kbd> or <kbd>Back</kbd> button on your mouse (if it have one) can also be used for that.

Arrow keys are used for panning images when zoom-in :)

(p.s. Please consider leaving a comment or vote on this thread if you also think using the arrow key to navigate images is important to you. I didn't add this feature just because I don't know the reason why it's better, I can also point out which line to edit so you can add this feature by yourself if needed)

**Reply by hardglitch** [reply]
OK. Thnx!

**Reply by BLumia** [reply]
Update:

I found there are actually a lot of people who prefer to use the Left/Right key to navigate between the previous and the next images, so I plan to add it as the default shortcut for image gallery navigation :)

It will be included in the next release (0.5.3), for users who would like to try it now, you can try the CI build now! For windows users you can get the CI build [here](https://ci.appveyor.com/project/BLumia/pineapplepictures/builds/41926133), for ubuntu 20.04 users you can get it [here](https://github.com/BLumia/pineapple-pictures/actions/runs/1590848004) (GitHub login required), or you can also pull the source code and build it manually :)

p.s. Originally, arrow keys are used for panning images when zoom-in. After this update, you can still pan images with keyboard-only by holding <kbd>Shift</kbd> and pressing arrow keys.
